go-controller/pkg/node/openflow_manager.go

```diff
@@ -219,7 +229,7 @@ func (c *openflowManager) updateBridgeFlowCache(subnets []*net.IPNet, extraIPs [
 	if err != nil {
 		return err
 	}
-	dftCommonFlows, err := commonFlows(subnets, c.defaultBridge, isPodNetworkAdvertised)
+	dftCommonFlows, err := commonFlows(subnets, c.defaultBridge, isPodNetworkAdvertised, isUDNNetworkAdvertised)
```

Comment:
We don't need to be passing `isPodNetworkAdvertised, isUDNNetworkAdvertised` around anymore, all that information is now readily available in `openflowManager.defaultBridge.netConfig[<network name>]`